### ISSUE TITLE
amcharts5 + vite 

### ISSUE THREAD

**Issue by cbroomemktw** [amcharts/amcharts5#1143]
**Bug description**

I was hung up by a nasty bug?, misconfiguration?, or similar when using amcharts5 with in a React app built with Vite. It seems one of amcharts dependencies `regression.js` is in a CommonJS format that Vite can't process.   I'm opening this ticket more to give a workaround for anyone else who may run into this. 

To fix, instruct Vite to exclude amcharts from optimization while also explicitly including regression as below: 

```
  optimizeDeps: {
    exclude: ['@amcharts/amcharts5'],
    include: ['@amcharts/amcharts5 > regression']
  },
```

This worked for me. Hope it works for you!

**Environment (if applicable)**

 - amCharts version 5.3.17
 - Chrome 118
 - vite 4.3.0 ~ 4.5.0

**Additional context**

**Comment by Pauan** [comment]
Thanks for the report. In the next version we have removed the `regression` package, so this won't be an issue anymore.

**Comment by martynasma** [comment]
Fixed in [5.5.5](https://github.com/amcharts/amcharts5/releases/tag/5.5.5).

## [5.5.5] - 2023-11-06

### Added
- SMA line added to Relative Strength Index indicator in `StockChart`.
- A possibility to draw a Parallel Channel added to `StockChart`.

### Changed
- Removed dependency on the `regression` npm package.

### Fixed
- Change of `minGridDistance` setting on `AxisRenderer` was not being applied until next redraw.
- Snapping of `StockChart` drawings sometimes worked incorrectly.
- Average line was drawn incorrectly on a `StockChart` (since `5.5.2`).

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
